### DISCUSSION TITLE
New forum icons aren't dark if dark theme is turned on.

### DISCUSSION THREAD

**Discussion by ItsEnderium** [ScratchAddons/ScratchAddons#2400] [Ideas]
I would like the new forum icons to turn dark if "Website dark theme" is on.

**Comment by TheColaber** [comment]
![image](https://user-images.githubusercontent.com/72760579/117296723-624a2400-ae43-11eb-9d6f-a82ef3a9239d.png)
These were simply `filter: invert(1);` but inverting scratch's default looks like this :/
![image](https://user-images.githubusercontent.com/72760579/117296813-7d1c9880-ae43-11eb-8268-0467c29a4389.png)

**Reply by GrahamSH-LLK** [reply]
That second one actually looks cool :eyes:
